src/cluster_legacy.c

```diff
@@ -148,6 +149,7 @@ static inline int defaultClientPort(void) {
     (server.cluster->slots[slot] == NULL || bitmapTestBit(server.cluster->owner_not_claiming_slot, slot))
 
 #define RCVBUF_INIT_LEN 1024
+#define RCVBUF_MIN_READ_LEN 16
```

Comment:
add a static assert to ensure `RCVBUF_MIN_READ_LEN` indeed covers the type/ver/len of the light msg hdr?

Reply:
I think we can get away with reading min 14
`#define RCVBUF_MIN_READ_LEN 14
static_assert(offsetof(clusterMsg, type) + sizeof(uint16_t) == RCVBUF_MIN_READ_LEN,
              "Incorrect lenght to read to identify type");`